settings.gradle.kts

```diff
@@ -129,6 +129,8 @@ include(":spi:transfer-spi")
 include(":spi:contract-spi")
 include(":spi:catalog-spi")
 
+//include(":openapi")
```

Comment:
For what do we need this?

Reply:
we don't. this is a remnant of a module that isn't there anymore :) will remove.